internal/core/application/service.go

```diff
@@ -2023,6 +2020,19 @@ func (s *service) GetInfo(ctx context.Context) (*ServiceInfo, errors.Error) {
 		}
 	}
 
+	// TODO: caching?
```

Comment:
in fact, we need to cache the whole info response and update it only when it is mutated ? let's open an issue @altafan?